modules/hal_logger/src/hal_log.hpp

```diff
@@ -0,0 +1,76 @@
+#ifndef HAL_LOG_HPP
```

Comment:
You might wanna move the include guard fully under the comment block instead of partly :p